app/addons/databases/actions.js

```diff
@@ -65,9 +65,9 @@ function (app, FauxtonAPI, Stores, ActionTypes, Resources) {
     },
 
     createNewDatabase: function (databaseName) {
-      if (_.isNull(databaseName) || databaseName.trim().length === 0) {
+      if (_.isNull(databaseName) || databaseName.trim().length === 0 || !this.isValidDatabaseName(databaseName)) {
         FauxtonAPI.addNotification({
-          msg: 'Please enter a valid database name',
+          msg: 'Please enter a valid database name. Only lowercase letters (a-z), digits (0-9), and any of the characters _, $, (, ), +, -, and / are allowed. Moreover, the database name must begin with a letter.',
```

Comment:
What about -

`Please enter a valid database name. The database must start with a letter and can only contain lowercase letters (a-z), digits (0-9) and the following characters _, $, (, ), +, -, and /.`